Source/Core/DolphinQt/Config/Graphics/EnhancementsWidget.cpp

```diff
@@ -421,14 +421,14 @@ void EnhancementsWidget::UpdateAAOptions()
   m_aa_combo->Reset();
   m_aa_combo->Add(tr("None"), (u32)1, false);
 
-  std::vector<u32> aa_modes = g_Config.backend_info.AAModes;
+  std::vector<u32> aa_modes = g_backend_info.AAModes;
```

Comment:
```suggestion
  const std::vector<u32>& aa_modes = g_backend_info.AAModes;
```

Not your code but..avoid a copy

Reply:
I'll save that for another PR.